app/com/linkedin/drelephant/analysis/AnalyticJobGeneratorHadoop2.java

```diff
@@ -46,10 +46,14 @@
   private static final String IS_RM_HA_ENABLED = "yarn.resourcemanager.ha.enabled";
   private static final String RESOURCE_MANAGER_IDS = "yarn.resourcemanager.ha.rm-ids";
   private static final String RM_NODE_STATE_URL = "http://%s/ws/v1/cluster/info";
+  private static final String FETCH_ONLY_RECENT_RM_APPS = "drelephant.analysis.only.fetchRecentApps";
```

Comment:
I believe that if a property is only used by Dr elephant and not by Hadoop, then that property should belong to Dr. Elephant configs, such as GeneralConf.